dangerzone/capture_output.py

```diff
@@ -0,0 +1,54 @@
+import io
+import logging
+import subprocess
+from typing import Any
+
+log = logging.getLogger()
+
+original_subprocess_run = subprocess.run
+
+
+def patched_subprocess_run(  # type: ignore[no-untyped-def]
+    args,
+    *,
+    stdout=None,
+    stderr=None,
+    capture_output=False,
+    **other_popen_kwargs,
+) -> subprocess.CompletedProcess:
+    # capture_output and stderr/stdout can't be specified at the same time
+    if not capture_output:
```

Comment:
In this case, if `capture_output=False` and `stdout=None` and `stderr=None`, the intent of the user is to print the output to the standard streams, potentially even live. However, I think the end result here is that the stdout/stderr will be logged only once the process finishes.

For instance, when `podman pull` runs:
* GUI users will see the output only once the process is over. Unfortunately, this does not make it a feasible progress bar alternative.
* CLI users will similarly not see any output, whereas they were able to see it in the past.